gems/sorbet-runtime/test/types/method_patches.rb

```diff
@@ -5,7 +5,11 @@ module Opus::Types::Test
   class MethodPatchesTest < Critic::Unit::UnitTest
     module MethodRefinement
       refine Method do
-        prepend(T::CompatibilityPatches::MethodExtensions)
+        if respond_to?(:refined_class)
+          refined_class.prepend(T::CompatibilityPatches::MethodExtensions)
```

Comment:
I don't think this will work the way you expect it to. The point of this being a refinement was so that it would not change the behaviour of `Method` when the refinement was not activated.

However, if the `prepend` is done against the refined class, then that will change the behaviour of that class for the rest of the test suite.

This needs a different approach.